ci-operator/step-registry/rosa/aws/sts/hypershift-sector/OWNERS

```diff
@@ -0,0 +1,16 @@
+reviewers:
+- heliubj18
+- LiangquanLi930
+- yasun1
+- xueli181114
+- yuwang-RH
+- tzhou5
+- yingzhanredhat
+- yufchang
```

Comment:
I can understand adding managers as approvers. But adding the manager as reviewer means that some PRs might be assigned to them (by the openshift-bot) to review the code changes. Is it expected ?  @yufchang @heliubj18

Reply:
good cache! I copied from existing rosa steps by @yasun1.  To make less noise, I will remove the manager from reviewer list. cc @yasun1